contact_energy_nz/usage_datum.py

```diff
@@ -12,7 +12,12 @@ def __init__(self, json_item) -> None:
             json_item["date"], "%Y-%m-%dT%H:%M:%S.%f%z"
         )
         self.value = float(json_item["value"])
-        self.dollar_value = float(json_item["dollarValue"])
+
+        try:
```

Comment:
wouldn't using `json_item.get("dollarValue", None)` here be more concise?

Reply:
well, in fact since I have opted for this pattern to begin with, I probably should retract that suggestion until I'm ready to refactor the whole class :)
thank you for the contribution 👍